tests/commands/dev/test_call.py

```diff
@@ -1,184 +1,217 @@
 import pytest
 
+from briefcase.commands import DevCommand
+from briefcase.commands.base import full_kwargs
 from briefcase.exceptions import BriefcaseCommandError
 
 
-def test_no_git(dev_command, first_app):
+class DummyDevCommand(DevCommand):
+    """
+    A dummy Dev command that doesn't actually do anything.
+    It only serves to track which actions would be performend.
+    """
+
+    platform = "tester"
+    output_format = "dummy"
+    description = "Dummy dev command"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, apps=[], **kwargs)
+
+        self.actions = []
+        self.env = dict(a=1, b=2, c=3)
+
+    def install_dev_dependencies(self, app, **kwargs):
+        self.actions.append(("dev_dependencies", app.app_name, kwargs))
+
+    def get_environment(self, app):
+        return self.env
+
+    def run_dev_app(self, app, env, **kwargs):
+        self.actions.append(("run_dev", app.app_name, kwargs, env))
+        return full_kwargs({"run_dev_state": app.app_name, "env": env}, kwargs)
+
+
+@pytest.fixture
+def dummy_dev_command(tmp_path):
+    return DummyDevCommand(base_path=tmp_path)
+
+
+def test_no_git(dummy_dev_command, first_app):
     "If Git is not installed, an error is raised"
     # Mock a non-existent git
-    dev_command.git = None
+    dummy_dev_command.git = None
```

Comment:
This is the code churn issue I was referring to. Patches should, generally, be as small as possible. Unless there's a good reason, arbitrary renames and restructures should be avoided. The problem I was flagging was the mass-rename of "dev_command" to "dummy_dev_command". That rename is a significant portion of the final diff, but doesn't actually improve any aspect of the patch.